apps/src/calc/calc.js

```diff
@@ -80,6 +81,12 @@ function getTokenList(one, two) {
   if (two instanceof Equation) {
     two = two.expression;
   }
+  if (jsnums.isSchemeNumber(one)) {
+    one = new ExpressionNode(one);
+  }
+  if (jsnums.isSchemeNumber(two)) {
+    two = new ExpressionNode(two);
+  }
   if (typeof(one) === 'string') {
```

Comment:
Marked is kind of confusing, since I've kind of overloaded the term/property for two different purposes. See the comment at the top of Token.js for an explanation.